Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathProjection.php

```diff
@@ -706,10 +705,24 @@ private function isNodeExplicitlyDisabled(DocumentNodeInfo $node): bool
         return $node->getDisableLevel() - $parentDisabledLevel !== 0;
     }
 
+    private function isSiteNodeType(NodeTypeName $nodeTypeName): bool
+    {
+        // HACK: We consider the currently configured node type of the given node.
+        // This is a deliberate side effect of this projector!
+        if (!$this->nodeTypeManager->hasNodeType($nodeTypeName)) {
+            return false;
+        }
+        $nodeType = $this->nodeTypeManager->getNodeType($nodeTypeName);
+        return $nodeType->isOfType(NodeTypeNameFactory::NAME_SITE);
```

Comment:
```suggestion
        return $nodeType->isOfType(NodeTypeNameFactory::forSite());
```

Reply:
changed for all usages of `NodeTypeNameFactory`